src/teller/teller.go

```diff
@@ -71,15 +73,16 @@ func (s *Teller) Shutdown() {
 
 // Service combines Exchanger and AddrGenerator
 type Service struct {
-	cfg       config.Teller
-	exchanger exchange.Exchanger  // exchange Teller client
-	addrGen   addrs.AddrGenerator // address generator
+	cfg        config.Teller
+	exchanger  exchange.Exchanger  // exchange Teller client
+	addrGen    addrs.AddrGenerator // address generator
+	ethAddrGen addrs.AddrGenerator // address generator
 }
 
 // BindAddress binds skycoin address with a deposit btc address
 // return btc address
 // TODO -- support multiple coin types
-func (s *Service) BindAddress(skyAddr string) (string, error) {
+func (s *Service) BindAddress(skyAddr, coinType string) (string, error) {
 	if s.cfg.MaxBoundBtcAddresses > 0 {
```

Comment:
We should rename this config to `MaxBoundAddresses` if we are applying the maximum limit to all types of addresses together

Reply:
ok, done